akka-remote/src/test/scala/akka/remote/RemotingSpec.scala

```diff
@@ -818,6 +820,31 @@ class RemotingSpec extends AkkaSpec(RemotingSpec.cfg) with ImplicitSender with D
 
     }
 
+    // retry a few times as the temporaryServerAddress can be taken by the time the new actor system
+    // binds
+    def selectionAndBind(
+        config: Config,
+        thisSystem: ActorSystem,
+        probe: TestProbe,
+        retries: Int = 3): (ActorSystem, ActorSelection) = {
+      val otherAddress = temporaryServerAddress()
+      val otherConfig = ConfigFactory.parseString(s"""
+              akka.remote.netty.tcp.port = ${otherAddress.getPort}
+              """).withFallback(config)
+      val otherSelection =
+        thisSystem.actorSelection(s"akka.tcp://other-system@localhost:${otherAddress.getPort}/user/echo")
+      otherSelection.tell("ping", probe.ref)
+      probe.expectNoMessage(1.seconds)
```

Comment:
this makes it more likely that the port is taken by someone else

Reply:
this isn't new, i think it is valid for the test to make sure it doesn't receive a response until the new system has bound. I've reduced it to 200ms